skills/csharp/experimental/itsmskill/Dialogs/CreateTicketDialog.cs

```diff
@@ -70,6 +81,31 @@ public class CreateTicketDialog : SkillDialogBase
             KnowledgeHelpLoop = Actions.DisplayExisting;
         }
 
+        protected override async Task<DialogTurnResult> OnBeginDialogAsync(DialogContext dc, object options, CancellationToken cancellationToken = default)
+        {
+            if (dc.Context.Activity.ChannelId.Contains("msteams"))
```

Comment:
please compare if equal to this https://github.com/microsoft/botbuilder-dotnet/blob/f0599ca72800c2b145ea7faed22d9bff8f297889/libraries/Microsoft.Bot.Connector/Channels.cs#L64

So if in Teams, we usually use TaskModule to create an incident? Is it the new design? Thanks